### ISSUE TITLE
[Bug Report] Selection controls hover color when false

### ISSUE THREAD

**Issue by MohawkMan** [vuetifyjs/vuetify#10979]
### Environment
**Vuetify Version:** 2.2.19
**Vue Version:** 2.6.11
**Browsers:** Chrome 80.0.3987.132
**OS:** Windows 10

### Steps to reproduce
Assign a color other than primary to a selection control. Hover over the control when it is not selected

### Expected Behavior
The hover color would be the color set on the control

### Actual Behavior
If not selected or checked the hover color is primary

### Reproduction Link
<a href="https://codepen.io/jonnymohawk/pen/MWwLZmW" target="_blank">https://codepen.io/jonnymohawk/pen/MWwLZmW</a>

### Other comments
I though it would have been fixed in issue 9431 but it hasn't been
https://github.com/vuetifyjs/vuetify/pull/9431

**Comment by billsliu** [comment]
This is fine, nobody will think this is an issue, I like the background color to show primary color

**Comment by Raja4567** [comment]
@billsliu I may be fine for your requirements. But for me, the client specifically chooses to use a different color.

**Comment by MohawkMan** [comment]
@billsliu obviously I created it as an issue so unless I am nobody... than someone things it is an issue.

@KaelWD I am not sure if I should create a new issue for this but this also seems to be affecting selects. I attached an image of a select with color set to accent and the label, icon and underline seem fine (colored accent) but the selected item in the menu is primary. 

![image](https://user-images.githubusercontent.com/9752771/78194037-db9d6900-7430-11ea-98aa-1ead7d12c338.png)

On a side note I didn't answer the question of if it worked before when creating the issue but it did. I am finding these issues as I am (finally) upgrading from 1.3
